app_spec/zomes/blog/code/src/memo.rs

```diff
@@ -47,7 +47,7 @@ pub fn definition() -> ValidatingEntryType {
         sharing: Sharing::Private,
 
         validation_package: || {
-            hdk::ValidationPackageDefinition::ChainFull
```

Comment:
@willemolding i'm kind of skim reading but don't these need to be `ChainFull` if we're testing full chain validation?

Reply:
These are private entries anyway (see line 47) so these are not actually being used to test the validation. These never needed to be ChainFull to begin with so I changed them to Entry to simplify debugging.

It is the `blog` entry types that are used for testing offline validation and these have the `ChainFull` validation package requirement.